blockbase/inc/customizer/wp-customize-fonts.php

```diff
@@ -209,13 +209,66 @@ class GlobalStylesFontsCustomizer {
 	);
 
 	function __construct() {
+		if ( $this->site_editor_is_implementing_google_fonts()  ) {
+			add_action( 'customize_register', array( $this, 'init_deprecation_notice' ) );
+			return;
+		}
+
 		add_action( 'customize_register', array( $this, 'initialize' ) );
 		add_action( 'customize_preview_init', array( $this, 'handle_customize_preview_init' ) );
 		add_action( 'customize_register', array( $this, 'enqueue_google_fonts' ) );
 		add_action( 'customize_save_after', array( $this, 'handle_customize_save_after' ) );
 		add_action( 'customize_controls_enqueue_scripts', array( $this, 'customize_control_js' ) );
 	}
 
+	function site_editor_is_implementing_google_fonts() {
+		$jetpack_has_google_fonts_module = false;
+		$gutenberg_webfonts_api_supports_enqueueing = false;
+
+		if ( defined( 'JETPACK__VERSION' ) ) {
+			$jetpack_has_google_fonts_module = JETPACK__VERSION === 'wpcom' || version_compare( JETPACK__VERSION, '10.8', '>' );
```

Comment:
**TODO:**
Remove unnecessary reference to `wpcom`. We control all versioning for Jetpack, Gutenberg, and jetpack modules for simple sites.